end-to-end-tests-localdb/docker/catalina_server.xml.patch

```diff
@@ -0,0 +1,11 @@
+--- server.xml.default	2016-07-18 15:10:55.000000000 +0200
++++ server.xml	2016-07-18 15:30:38.000000000 +0200
+@@ -67,6 +67,8 @@
```

Comment:
this u don't need with the new Docker setup

Reply:
File was removed.